agent/agent.go

```diff
@@ -648,6 +648,7 @@ func (a *agent) createTailnet(ctx context.Context, derpMap *tailcfg.DERPMap) (_
 				}
 				break
 			}
+			logger.Debug(ctx, "accepted pty conn", slog.F("remote", conn.RemoteAddr().String()))
```

Comment:
```suggestion
			logger.Debug(ctx, "accepted conn", slog.F("remote", conn.RemoteAddr().String()))
```
Logger name already contains pty.